src/main/java/hudson/scm/listtagsparameter/ListSubversionTagsParameterDefinition.java

```diff
@@ -171,14 +171,15 @@ public DescriptorImpl getDescriptor() {
   @Nonnull public List<String> getTags(@Nullable Job context) {
     List<String> dirs = new ArrayList<String>();
 
+    SVNRepository repo = null;
```

Comment:
Better style (IMO) to just have a nested `try`-`finally` block with `repo` as an effectively-`final` variable. Would migrate nicely to `-source 7`, assuming `SVNRepository` is `AutoCloseable`.